internal/ask/form/form.go

```diff
@@ -102,6 +102,8 @@ func Upsert(context interface{}, db *db.DB, form *Form) error {
 	if form.ID.Hex() == "" {
 		form.ID = bson.NewObjectId()
 		isNewForm = true
+		form.DateCreated = time.Now()
+		form.DateUpdated = time.Now()
 	}
```

Comment:
@wyattjoh so should I just move the DateUpdated outside the if or also remove it from UpdateStats?